src/pages/signup.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBCard,
+  MDBCardBody,
+  MDBInput,
+  MDBBtn,
+} from 'mdbreact';
+import './signup.scss';
+
+function SignPage() {
```

Comment:
It should have a better more explaining name
```suggestion
function SignUpPage() {
```